From 770e22521ce09a6440910b475f99745d86d95140 Mon Sep 17 00:00:00 2001
From: Ikey Doherty <ikey@solus-project.com>
Date: Sun, 1 Jan 2017 14:14:24 +0000
Subject: [PATCH] schemas: Add overrides for Brisk Menu

This override will have the Solus Software Center displayed in the
shortcuts sidepane of the Brisk Menu by default.

Signed-off-by: Ikey Doherty <ikey@solus-project.com>
---
 schemas/mate/Makefile.am                                   | 3 ++-
 schemas/mate/com.solus-project.brisk-menu.gschema.override | 2 ++
 2 files changed, 4 insertions(+), 1 deletion(-)
 create mode 100644 schemas/mate/com.solus-project.brisk-menu.gschema.override

diff --git a/schemas/mate/Makefile.am b/schemas/mate/Makefile.am
index a411973..0185f6f 100644
--- a/schemas/mate/Makefile.am
+++ b/schemas/mate/Makefile.am
@@ -8,7 +8,8 @@ MATE_SCHEMAS = \
 	org.mate.panel.gschema.override \
 	org.mate.panel.applet.clock.gschema.override \
 	org.mate.panel.menubar.gschema.override \
-	org.mate.NotificationDaemon.gschema.override
+	org.mate.NotificationDaemon.gschema.override \
+	com.solus-project.brisk-menu.gschema.override
 
 %.gschema.override:
 	rm -f $@ ;
diff --git a/schemas/mate/com.solus-project.brisk-menu.gschema.override b/schemas/mate/com.solus-project.brisk-menu.gschema.override
new file mode 100644
index 0000000..5bf3210
--- /dev/null
+++ b/schemas/mate/com.solus-project.brisk-menu.gschema.override
@@ -0,0 +1,2 @@
+[com.solus-project.brisk-menu]
+pinned-shortcuts=['matecc.desktop', 'solus-sc.desktop']
-- 
2.11.0

